django/db/backends/sqlite3/schema.py

```diff
@@ -507,6 +520,14 @@ def _alter_many_to_many(self, model, old_field, new_field, strict):
                         new_field.m2m_reverse_field_name()
                     ),
                 ),
+                alter_self_field=(
```

Comment:
Have you seen the @andrewgodwin comment above? (see 03900a02d5fd2efd6ae98bdb9974b78146f63040)

> _The field that points to the target model is needed, so_
> _we can tell alter_field to change it - this is_
> _m2m_reverse_field_name() (**as opposed to m2m_field_name()**,_
> _which points to our model)._

I'm a little worried that we're missing something here :thinking: We should at least by update this comment.